desktop_version/src/Font.cpp

```diff
@@ -16,7 +17,14 @@ SDL_Texture* LoadImage(const char *filename, const TextureLoadType loadtype);
 namespace font
 {
 
-Font temp_bfont; // replace with like, a vector of all loaded fonts
+static FontContainer fonts_main = {0};
+static FontContainer fonts_custom = {0};
```

Comment:
Hm, I don't remember ever seeing that warning, and I'm not seeing it now on my Linux GCC. I do think I wanted to be very explicit about the zero-initialization here, hence the extra 0. I'll try to see which CIs give the warning and whether `{}` resolves it (otherwise I guess not being explicit about it is not the end of the world, might just cause someone to think it's uninitialized data that'll wreak havoc).

Reply:
It's not on CI. I compile with `-Wall -Wextra`.

Reply:
Oh, I always thought that was the default in our CMakeLists.